### ISSUE TITLE
x.sum('field').then(console.log) triggers 'int' object has no attribute '__getitem__' error

### ISSUE THREAD

**Issue by ptitzler** [ibm-watson-data-lab/pixiedust_node#27]
PD 1.1.7 pd_node 0.2.3

Didn't try other aggregation operators.
```
%%node
var cqs = require('cloudant-quickstart');
const cities = cqs('https://56953ed8-3fba-4f7e-824e-5498c8e1d18e-bluemix.cloudant.com/cities');
cities.get('2636749').then(print).catch(console.error);
// works
cities.get('2636749').then(console.log).catch(console.error);
// works
cities.sum('population').then(print).catch(print);
2694222973
// fails
cities.sum('population').then(console.log).catch(print);

2694222973
'int' object has no attribute '__getitem__'
```

Workaround: use `print` instead of `console.log` or `console.error`

Source notebook: https://github.com/ibm-watson-data-lab/nodebook-code-pattern/blob/master/notebooks/nodebook_1.ipynb

**Comment by glynnbird** [comment]
Swap `var cqs` for `const cqps`.

**Comment by ptitzler** [comment]
I've tried that but the issue remained.

**Comment by glynnbird** [comment]
Found the problem. If you `console.log` a single integer from a node cell, then the pixiedust_node parser picks it up and examines it to see if it's a "Node-->Python" variable message. (it gets through because an integer on its own parses as JSON correctly). I've put in extra checks so that only dictionaries get through.

**Comment by glynnbird** [comment]
Fixed in 0.2.4
